bumble/transport/common.py

```diff
@@ -257,13 +259,6 @@ def on_transport_lost(self) -> None:
             if hasattr(self.parser.sink, 'on_transport_lost'):
                 self.parser.sink.on_transport_lost()
 
-    async def wait_for_termination(self) -> None:
```

Comment:
It's too early to remove this method, it is still being used by a lot of code.

Reply:
Reverted